frontend/src/components/7.Result/ResultPage.js

```diff
@@ -110,41 +123,171 @@ function ResultPage(props) {
+                <Card.Body
+                  style={{
+                    display: "flex",
+                    alignItems: "center",
+                    justifyContent: "center",
+                  }}
+                >
+                  <Row
+                    lg={12}
+                    style={{
+                      display: "flex",
+                      alignItems: "center",
+                      justifyContent: "center",
+                      maxWidth: "95%",
+                    }}
+                    className="justify-content-md-center"
+                  >
+                    <Col
+                      style={{
+                        display: "flex",
+                        alignItems: "center",
+                        justifyContent: "center",
+                        maxHeight: "50%",
+                        marginTop: "1em",
+                      }}
+                    >
+                      <SwipeCard class="text-primary" data={data} />
+                    </Col>
+                    <Col
+                      style={{
+                        display: "flex",
+                        alignItems: "center",
+                        justifyContent: "center",
+                      }}
+                      lg={12}
+                    >
+                      <Button
+                        onClick={() => setResultPopup(true)}
+                        className="BigBtn"
+                        id="Resultbutton"
+                        style={{
+                          fontSize: "2em",
+                          marginTop: "0.1em",
+                          marginBottom: "0.1em",
+                        }}
+                      >
+                        See results
+                      </Button>
+
+                      <Result trigger={ResultPopup} setTrigger={setResultPopup}>
+                        {chart && (
+                          <DataVisual className="DataVisual" data={pie} />
+                        )}
+                        <hr />
+                      </Result>
+                    </Col>
+
+                    <Col
+                      style={{
+                        display: "flex",
+                        alignItems: "center",
+                        justifyContent: "center",
+                      }}
+                      lg={12}
+                    >
+                      <Button
+                        onClick={() => setMapPopup(true)}
+                        className="BigBtn"
+                        id="GoogleMaps_btn"
+                        style={{ marginTop: "0.15em", marginBottom: "0.15em" }}
+                      >
+                        View on Google Maps
+                        <Icon />
+                      </Button>
+                      <MapModal
+                        trigger={MapPopup}
+                        setTrigger={setMapPopup}
+                        restaurantLocation={{ lat: -36.8523, lng: 174.7691 }}
```

Comment:
could you please change this line to `restaurantLocation={data.coords}`

Reply:
done